autogluon/task/tabular_prediction/predictor.py

```diff
@@ -341,6 +341,64 @@ def fit_summary(self, verbosity=3):
             print("*** End of fit() summary ***")
         return results
 
+    def transform_features(self, dataset=None, model=None, base_models=None):
+        """
+        Transforms dataset features through the AutoGluon feature generator.
+        This is useful to gain an understanding of how AutoGluon interprets the dataset features.
+        The output of this function can be used to train further models, even outside of AutoGluon.
+        When `dataset=None`, `base_models=[{best_model}], and bagging was enabled during fit():
+            This returns the out-of-fold predictions of the best model, which can be used as training input to a custom user stacker model.
+
```

Comment:
Include small example in the documentation? 
This function is a bit opaque without an example.

Reply:
Added